ModAssistant/Classes/Utils.cs

```diff
@@ -385,14 +385,14 @@ public static string GetManualFile(string filter = "", string title = "Open File
         public static bool IsVoid()
         {
             string directory = App.BeatSaberInstallDirectory;
-            string pluginsDirectory = Path.Combine(directory, "Beat Saber_Data", "Plugins");
+            string pluginsDirectory = Path.Combine(directory, "Beat Saber_Data", "Plugins", "x86_64"); // newer version of bs use x86_64 for plugins folder
```

Comment:
Instead of changing the plugins directory it would be best if ModAssistant checks both, since it can be used on older versions of the game too.